apps/email/style/common.css

```diff
@@ -330,33 +337,34 @@ the list of folders.
 */
 .email-value-selector[role="dialog"][data-type="value-selector"] [role="listbox"] .fld-folder-depth0,
 .fld-folder-depth0 {
-  -moz-padding-start: 1.5rem;
+  padding-inline-start: 1.5rem;
 }
 .email-value-selector[role="dialog"][data-type="value-selector"] [role="listbox"] .fld-folder-depth1,
 .fld-folder-depth1 {
-  -moz-padding-start: 3rem;
+  padding-inline-start: 3rem;
 }
 .email-value-selector[role="dialog"][data-type="value-selector"] [role="listbox"] .fld-folder-depth2,
 .fld-folder-depth2 {
-  -moz-padding-start: 4.5rem;
+  padding-inline-start: 4.5rem;
 }
 .email-value-selector[role="dialog"][data-type="value-selector"] [role="listbox"] .fld-folder-depth3,
 .fld-folder-depth3 {
-  -moz-padding-start: 6rem;
+  padding-inline-start: 6rem;
 }
 .email-value-selector[role="dialog"][data-type="value-selector"] [role="listbox"] .fld-folder-depth4,
+
 .fld-folder-depth4 {
-  -moz-padding-start: 6.5rem;
+  padding-inline-start: 7.5rem;
```

Comment:
I've opened a bug some time about folder depth: https://bugzilla.mozilla.org/show_bug.cgi?id=1169589

Part of it got corrected, but comment 12 asked me to open another bug for the css part. That's the fix for this bug I've never opened :-)

For the reference, here are some screenshots

**Before**
LTR - RTL 

![drawer_02](https://cloud.githubusercontent.com/assets/1576598/10695122/5555ccbc-79a2-11e5-8ed6-902e5f8477ce.png)

**After**
LTR - RTL

![drawer_02](https://cloud.githubusercontent.com/assets/1576598/10695130/5da34f84-79a2-11e5-8653-fc2730927714.png)